artiq/gateware/eem.py

```diff
@@ -757,3 +758,33 @@ def add_std(cls, target, eem, ttl_out_cls, iostandard=default_iostandard):
             phy = ttl_out_cls(pads.p, pads.n)
             target.submodules += phy
             target.rtio_channels.append(rtio.Channel.from_phy(phy))
+
+
+class FMCCarrier(_EEM):
+    @staticmethod
+    def io(eem, role="master", iostandard=default_iostandard):
```

Comment:
It is the DRTIO role: when it is a DRTIO master, the `role` argument shall be `"master"`, satellite otherwise. The `role` parameter is solely for assigning the corresponding EEM I/O pair.